Phar/Phar.php

```diff
@@ -73,7 +73,10 @@ public function __destruct() {}
      * @return void no return value, exception is thrown on failure.
      */
     #[TentativeType]
-    public function addEmptyDir(#[LanguageLevelTypeAware(['8.0' => 'string'], default: '')] $directory): void {}
+    public function addEmptyDir(
+        #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] $directory = '',
```

Comment:
Sorry, is there a default value for `$directory`? I've checked that with https://3v4l.org/9fb0m and looks like there isn't.
Based on this, it looks like parameter is only available in version 7.3-7.4 and > 8.0.

The signature looks like this for me:
```
    public function addEmptyDir(
        #[PhpStormStubsElementAvailable(from: '7.3', to: '7.4')] $dirname,
        #[PhpStormStubsElementAvailable(from: '8.0')] #[LanguageLevelTypeAware(['8.0' => 'string'], default: '')] $directory
    )
```
Let me know if I'm reading that wrong

Reply:
Edit: please disregard, I've run it only on supported versions. Will double-check

Reply:
So, taking previous 3v4l result in mind, the signature would look like
```
    public function addEmptyDir(
        #[PhpStormStubsElementAvailable(from: '5.3', to: '7.4')] #[LanguageLevelTypeAware(['7.4' => 'string'], default: '')] $dirname,
        #[PhpStormStubsElementAvailable(from: '8.0')] string $directory
    )
```
instead?

Reply:
Updated to
```
#[TentativeType]
    public function addEmptyDir(
        #[PhpStormStubsElementAvailable(from: '5.3', to: '7.4')] $directory = '',
        #[PhpStormStubsElementAvailable(from: '8.0')] string $directory
    ): void {}
```